### ISSUE TITLE
Server failed to start, EULA didn't pop up

### ISSUE THREAD

**Issue by flareofghast** [hexparrot/mineos-node#59]
So, I was just testing the first profile not showing up issue (I deleted /var/games/minecraft) restarted the service, downloaded the 1.8.7 profile, created the test server waited like 30 seconds and it still didn't show. I restarted the mineos service, refreshed the webui page and relogged in. The server was listed, I then clicked on it and picked the profile, change runnable jar to, 1000 1000 for xms xmx. Clicked start - up came your pop up which prompted me to pick the profile, runnable jar and xmx xms, which had maintained my settings so no need to do anything, click OK or whatever and then the server failed to start.
No EULA pop up.
Click the stop button arrow, select accept eula and click start and failed. Nothing helpful about why it failed in the log or webui console when started sudo node webui.js.

info: [test] received request "start"
error: [test] command "start" errored out: command=start, uuid=dbf71a80-1dec-11e5-8d0a-2dcc1a1a01ba, time_initiated=1435533884456, success=false, err=1, time_resolved=1435533884616

also saw
````
sudo node webui.js 
MineOS webui listening on *:8443
info: [test] Discovered server
error: [test] Create tail on logs/latest.log failed
info: [test] Watching for file generation: logs/latest.log
error: [test] Create tail on server.log failed
info: [test] Watching for file generation: server.log
````

**Comment by flareofghast** [comment]
with the new rsync I have not yet had an issue.

**Comment by flareofghast** [comment]
sorry, not the not starting issue, the first profile creation one still exists

**Comment by hexparrot** [comment]
So I have a full picture of what you're working with.  What distro are you installing this on?

Also, to clarify, installing the recent rsync allowed you to *start* the server, correct?

Finally, I also want to clarify 'the first profile creation one' problem you mean 'first server', right?  Because you shouldn't have to create profiles anymore (simply download them), and what's happening is after you create a server, it doesn't immediately appear on the dashboard.

Ok, looking into it.

**Comment by flareofghast** [comment]
Centos 7 at the moment.

with the new rsync I haven't had the no eula pop up with vanilla. I did have a no EULA with direwolf.

yes I meant first server.

**Comment by hexparrot** [comment]
Was the vanilla server > 1.7.9?

**Comment by flareofghast** [comment]
pretty sure it was 1.8.7

**Comment by hexparrot** [comment]
Can't seem to reproduce this with the appropriate vanilla server versions; I'll keep an eye out, though.

**Comment by hexparrot** [comment]
I'm hoping that all EULA related issues should now be resolved, with the most recent #75 rewrite on detecting the eula state.  Confirm if it's still an issue, if possible.
